From edcfba50b5b62689a361127a42b273951d7eb239 Mon Sep 17 00:00:00 2001
From: simisimon <Bastisimon95@googlemail.com>
Date: Fri, 18 Mar 2022 13:59:24 +0100
Subject: [PATCH 7/8] Add pom.xml

---
 pom.xml | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 pom.xml

diff --git a/pom.xml b/pom.xml
new file mode 100644
index 0000000..f9e682f
--- /dev/null
+++ b/pom.xml
@@ -0,0 +1,4 @@
+<project>
+  <version>0.2.0</version>
+  <port>7777</port>
+</project>
\ No newline at end of file
-- 
2.25.1

